homeassistant/util/ssl.py

```diff
@@ -1,12 +1,70 @@
 """Helper to create SSL contexts."""
 import contextlib
+from functools import lru_cache
 from os import environ
 import ssl
 
 import certifi
 
+from homeassistant.backports.enum import StrEnum
 
-def create_no_verify_ssl_context() -> ssl.SSLContext:
+
+class SslCipherList(StrEnum):
+    """SSL cipher lists."""
+
+    DEFAULT = "default"
```

Comment:
This one might be confusing because you can do `sslcontext.set_ciphers("DEFAULT")` as well and it doesn't mean the same thing

Reply:
I guess we could call this `PYTHON_DEFAULT` unless you can think of something better.

Naming is hard 😉

Reply:
after a few rounds of reflection and a beer, I also came up with nothing better than `PYTHON_DEFAULT` 🙈